neofetch

```diff
@@ -849,11 +849,15 @@ getcpu () {
             speed=${speed:0:1}.${speed:1}
 
             cpu="$cpu @ ${speed}GHz"
+
```

Comment:
Remove the empty line and the `# Get cpu cores` line as the cores var explains itself clear enough.
eg.
```sh
cpu="$cpu @ ${speed}GHz"
cores=$(awk -F ': ' '/siblings/ {printf $2; exit}' /proc/cpuinfo)
```